### PR TITLE
Bring cl example mod in parity with standard example mod

### PR DESCRIPTION
**Pull request by triphora**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -49,43 +36,76 @@ minecraft {
...
 	// API. You technically don't need it, but it's extremely useful for not having to write the same code in every mod.
 	modImplementation "io.github.minecraft-cursed-legacy:cursed-legacy-api:${project.api_version}"
+
+	// PSA: Some older mods, compiled on Loom 0.2.1, might have outdated Maven POMs.
+	// You may need to force-disable transitiveness on them.
 }
 
 processResources {
-	inputs.property 'version', project.version
+	inputs.property "version", project.version
```
Comment (halotroop2288) [inline build.gradle:54]: Pointless change. This is a string literal. Has no inline variables. Single quotes are correct here.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Mon Aug 22 17:36:22 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```
Comment (halotroop2288) [inline gradle/wrapper/gradle-wrapper.properties]: In the interest of using purely Java 8, you should probably keep to Gradle 6.9.x.

File: src/main/resources/fabric.mod.json [hunk 3]
```diff
@@ -27,10 +27,10 @@
   ],
 
   "depends": {
-    "fabricloader": "*",
```
Comment (halotroop2288) [inline src/main/resources/fabric.mod.json:30]: Fabric loader is still going to be required. I don't see a point in removing it.

File: src/main/resources/fabric.mod.json [hunk 4]
```diff
@@ -27,10 +27,10 @@
...
-    "minecraft": "1.0.0-beta.7.3"
+    "minecraft": "1.0.0-beta.7.3",
+    "api": "*"
```
Comment (halotroop2288) [inline src/main/resources/fabric.mod.json:31]: Example mod shouldn't depend on the full API, just have it available.

File: README.md [hunk 5]
```diff
@@ -25,4 +25,5 @@ Once your mod is complete and ready for use it can be compiled and reobfuscated
 ```
 
 ## License
-This template is available under the MIT license. Feel free to learn from it and incorporate it in your own projects.
+
+This template is available under the CC0 license. Feel free to learn from it and incorporate it in your own projects.
```
Comment (halotroop2288) [inline README.md:29]: Don't relicense the project. If you intended to, you forgot the LICENSE file anyway.

File: build.gradle [hunk 6]
```diff
@@ -49,43 +36,76 @@ minecraft {
...
 	mappings "io.github.minecraft-cursed-legacy:plasma:b1.7.3-build.${project.plasma_build}"
-
 	modImplementation("io.github.minecraft-cursed-legacy:cursed-fabric-loader:${project.loader_version}") {
 		transitive false
 	}
 
 	// API. You technically don't need it, but it's extremely useful for not having to write the same code in every mod.
 	modImplementation "io.github.minecraft-cursed-legacy:cursed-legacy-api:${project.api_version}"
+
+	// PSA: Some older mods, compiled on Loom 0.2.1, might have outdated Maven POMs.
```
Comment (halotroop2288) [inline build.gradle:49]: Is this even applicable? We still use a modified version of Loom 0.2.

**Review by halotroop2288 — CHANGES_REQUESTED** [review]

File: src/main/resources/fabric.mod.json (hunk 3 above)
Comment (triphora) [reply src/main/resources/fabric.mod.json:30]: This file won't be loaded without Fabric Loader. I don't see a point in having it.

File: src/main/resources/fabric.mod.json (hunk 4 above)
Comment (triphora) [reply src/main/resources/fabric.mod.json:31]: Standard example mod depends on Fabric API

File: README.md (hunk 5 above)
Comment (triphora) [reply README.md:29]: The license file and fmj are already CC0, it's only the readme that says MIT

File: build.gradle (hunk 1 above)
Comment (triphora) [reply build.gradle:54]: It's what the standard example mod has, I just copy-pasted its buildscript and then manually put in the cl example mod changes

File: build.gradle (hunk 6 above)
Comment (triphora) [reply build.gradle:49]: Loom 0.2.x doesn't follow SemVer, each patch version increment in 0.2 should instead be considered as a minor version increment for practical purposes

Comment (halotroop2288) [reply build.gradle:49]: Oh yikes. OK whatever.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (triphora) [reply gradle/wrapper/gradle-wrapper.properties]: Done in 05982a1accc1c5cf88850673cb6244e36d53789f

**Review by halotroop2288 — APPROVED** [review]

**Review by valoeghese — APPROVED** [review]
Seems good

**Pull request merged**